bob_igbarumah/assignments/CoinToss/CoinToss.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+def toss():
+    head = 1
+    tail = 0
+    countH = 0
+    countT = 0
+
+    for i in range(1,5001):
+        x = random.random()
+        x_round = round(x)
+        if x_round == head:
```

Comment:
No need to define head as 1 and tail as 0.  You can just directly do your if check for that number (if x_round ==1: countH += 1)